Automattic/wp-calypso
# Store: Hook up product category saving

This PR adds some Redux code for updating a product category (creation code already existed). It also hooks up both actions to the UI, enabling saving for both new categories and existing categories.

Parent PR: #21351.
See: #20295.

To Test:
* Run `npm run test-client client/extensions/woocommerce/state` and make sure all tests pass.
* Go to `http://calypso.localhost:3000/store/products/category/:site` and test creating a new category.
* Select an existing category from the list `http://calypso.localhost:3000/store/products/categories/:site` and make an update to it.
